lib/db/instance.go

```diff
@@ -209,14 +205,18 @@ func (db *instance) withAllFolderTruncated(folder []byte, fn func(device []byte,
 	}
 }
 
-func (db *instance) getFile(key []byte) (protocol.FileInfo, bool) {
-	if f, ok := db.getFileTrunc(key, false); ok {
+func (db *instance) getFileDirty(folder, device, file []byte) (protocol.FileInfo, bool) {
+	return db.getFileDirtyByKey(db.keyer.GenerateDeviceFileKey(nil, folder, device, file))
+}
+
+func (db *instance) getFileDirtyByKey(key []byte) (protocol.FileInfo, bool) {
+	if f, ok := db.getFileDirtyTrunc(key, false); ok {
 		return f.(protocol.FileInfo), true
 	}
 	return protocol.FileInfo{}, false
 }
 
-func (db *instance) getFileTrunc(key []byte, trunc bool) (FileIntf, bool) {
+func (db *instance) getFileDirtyTrunc(key []byte, trunc bool) (FileIntf, bool) {
```

Comment:
There are now only calls to `db.getFileDirty(...` left, so this function and `getFileDirtyByKey` could be removed.